### ISSUE TITLE
Missing directory

### ISSUE THREAD

**Issue by gitxav** [DenzelCode/nest-auth#4]
Hi:
It seems a directory is missing. Running npm run start:dev I get:
```
Error: ENOENT: no such file or directory, open '~/Documents/Nestjs/nest-auth/secrets/apple-key.p8'
    at Object.openSync (fs.js:476:3)
    at Object.readFileSync (fs.js:377:35)
    at Object.<anonymous> (~/Documents/Nestjs/SELECCION-ULTIMA/nest-auth/src/features/auth/service/apple-auth.service.ts:16:20)
```
Best regards.

**Comment by DenzelCode** [comment]
You have to put the Key of your apple app in there, just create an empty file named like that in that directory

**Comment by DenzelCode** [comment]
Fixed on commit 6d3d8f8be237d82da274350cd3c4d7dc31e22b69
